.eslintrc

```diff
@@ -27,12 +27,12 @@
   "rules": {
     /* || Eslint main rules || */
     "camelcase": [2, { "properties": "always" }], // Override airbnb style.
-    "func-names": 2,
+    // "func-names": 2,
     "no-console": 2,
-    "no-unused-vars": [
-      2,
-      { "args": "after-used", "argsIgnorePattern": "^_", "vars": "local" }
-    ],
+    // "no-unused-vars": [
+    //   2,
+    //   { "args": "after-used", "argsIgnorePattern": "^_", "vars": "local" }
+    // ],
```

Comment:
What are the errors that are preventing this rule from staying on?

Reply:
This rule was commented out here because it was added to the audit on line 122

<img width="828" alt="Screen Shot 2020-05-22 at 9 46 39 AM" src="https://user-images.githubusercontent.com/55560129/82674361-2bb6d180-9c11-11ea-93a5-ef7332e52c6b.png">